src/Uno.Extensions.Navigation.UI/Navigator.cs

```diff
@@ -531,6 +531,18 @@ protected virtual async Task<bool> RegionCanNavigate(Route route, RouteInfo? rou
 			}
 		}
 
+		if (!(Route?.IsEmpty()??true))
```

Comment:
I think it's a little bit more readable.

```suggestion
		if (Route?.IsEmpty() == false)
```